docs/dev/reference/services.md

```diff
@@ -136,6 +136,49 @@ class Example
 ```
 
 
+## SimpleTokenParser
+
```

Comment:
As mentioned by @Toflar , pls add `{{< version "4.10" >}}` here :)

Reply:
Done in 0bce42b6fb9e29add028c3d63e072499dec33db6